CATALOG.md

```diff
@@ -0,0 +1,42 @@
+# Build Commands Library
+
+Below you'll find a list of build commands contributed from the nativefier community.
+
+If you'd like to add to this catalog, please keep the following in mind:
+
+Library commands should be the strict necessary to make an app work. So, for example,
+
+- Yes to have `--widevine` in Udemy. Yes to `--internal-urls` and `--browserwindow-options` for Outlook Web. Yes to a name and icon for both...
+- ... but let's not have all the other flags that are a matter of personal preference (e.g. `--disable-dev-tools` or `--disk-cache-size`).
+
+* * *
+
+## Outlook Web
+
+### Windows Command
+```
+nativefier https://outlook.office.com/mail 
+--name OutlookWA 
+--internal-urls ".*?(outlook.live.com|outlook.office365.com|outlook.office.com|login.microsoftonline.com).*?" 
```

Comment:
I'd say that `login.microsoftonline.com` should be removed here, and probably just added to the internal login page list, but that could probably be covered in a separate PR.

Reply:
I will remove... not sure I know what the internal login page list is tho...

Reply:
It's in app/src/helpers/helpers.ts (and there are tests in app/src/helpers/helpers.test.ts

Again, we can handle this in a separate PR.